recipes/pkg-config/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: pkg-config
+  version: 0.28
+
+source:
+  fn: pkg-config-0.28.zip
+  url: https://github.com/tpn/pkg-config/archive/pkg-config-0.28.zip
+
+build:
+  number: 2
+  detect_binary_files_with_prefix: 1
```

Comment:
Just to aid readability, could you change this to true? http://conda.pydata.org/docs/building/meta-yaml.html#detect-binary-files-with-prefix